.github/workflows/static_analysis.yaml

```diff
@@ -149,8 +149,10 @@ jobs:
           echo "*** Current Output of cmake_min_version.py ***"
           echo $output
           
+          wrong_output = "WRONG STRING"
+          
           # Retrieve CMake minimal version from the last line of the tool output.
-          current_cmake_version=$(echo ${output} | grep -oP "VERSION \d+\.\d+")
+          current_cmake_version=$(echo ${wrong_output} | grep -oP "VERSION \d+\.\d+")
```

Comment:
@maloel 
Yes, you right, grep failed the GHA test because didn't find substring according given pattern...
But there was not printed output why the failed happened only one line:
`Error: Process completed with exit code 127.`

So, I think it will helpful to leave check with regex pattern.

Reply:
Grep output was upgraded.